bip-0155.mediawiki

```diff
@@ -134,7 +134,7 @@ See the appendices for the address encodings to be used for the various networks
 
 Introduce a new message type <code>sendaddrv2</code>. Sending such a message indicates that a node can understand and prefers to receive <code>addrv2</code> messages instead of <code>addr</code> messages. I.e. "Send me addrv2".
 
-<code>sendaddrv2</code> SHOULD be sent after receiving the <code>verack</code> message from the peer.
+<code>sendaddrv2</code> SHOULD be sent before sending the <code>verack</code> message to the peer.
```

Comment:
```suggestion
<code>sendaddrv2</code> MUST be sent before sending the <code>verack</code> message to the peer.
```

I'd say to make the bip strict here. Otherwise the change doesn't have any benefit, as is simply says "the message *should* be sent before verack, but can actually be sent any time", which was true before already.

Reply:
Changed to `MUST` and reworded to mimic BIP-339 / wtxidrelay.